### ISSUE TITLE
Critical Error when one of the users tries to use the search bar

### ISSUE THREAD

**Issue by DrazorV** [manyfold3d/manyfold#3331]
**Describe the bug**
When one of my users uses the search bar a postgress error prevents him to see any results. It doesn't happen when I try to search for the same query. He tried from his desktop from chromium browser and firefox, it happens on both.

Here are all the logs I could find.

[Manyfold-DB_logs.txt](https://github.com/user-attachments/files/18222565/Manyfold-DB_logs.txt)
[Manyfold_logs.txt](https://github.com/user-attachments/files/18222566/Manyfold_logs.txt)
[error_report_2024-12-21.csv](https://github.com/user-attachments/files/18222567/error_report_2024-12-21.csv)
[slow_requests_report_2024-12-21.csv](https://github.com/user-attachments/files/18222568/slow_requests_report_2024-12-21.csv)

Also I have uploaded my slow requests to investigate why some queries take up to 1.5s to return results from the db, which adds up to the overall responsiveness of the site since my slow upload speed (20 mpbs) doesn't help a lot. 

(Don't forget, I have a huge library)
![image](https://github.com/user-attachments/assets/c347f49f-18a0-483e-8216-072b6497c15b)
![image](https://github.com/user-attachments/assets/a4b5405c-2996-4109-8a7c-e5594fff41ad)

**Comment by Floppy** [comment]
Is the user a contributor, or some other role? It might be the same problem as #3090, which has been nagging at me for ages.

**Comment by DrazorV** [comment]
> Is the user a contributor, or some other role? It might be the same problem as #3090, which has been nagging at me for ages.

No, he is a member, but he used to be a contributor in the past.

**Comment by Floppy** [comment]
Ok, thanks. Not an admin or moderator. That's probably relevant then.
